docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/amazon-quicksight.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Amazon QuickSight
```

Comment:
I would suggest the following structure based on priority for the feature understanding:

1. Analyses and dashboards
2. User management and permissions
3. Data sources and datasets
4. Resetting analytics

Reply:
We need to add a new section "Exporting data" according to the order above.

There, we need to describe the following use cases:

1. Exporting data from visuals
https://docs.aws.amazon.com/quicksight/latest/user/exporting-data.html

2. Exporting analyses or dashboards as PDFs
https://docs.aws.amazon.com/quicksight/latest/user/export-dashboard-to-pdf.html